test/integration/Windows/ImagesComparisonTest.cs

```diff
@@ -19,11 +19,13 @@ public void BeforeAll()
             appCapabilities.App = "Microsoft.WindowsCalculator_8wekyb3d8bbwe!App";
             appCapabilities.DeviceName = "WindowsPC";
             appCapabilities.PlatformName ="Windows";
+            appCapabilities.AutomationName= "Windows";
 
             var serverUri = Env.ServerIsRemote() ? AppiumServers.RemoteServerUri : AppiumServers.LocalServiceUri;
             _calculatorSession = new WindowsDriver(serverUri, appCapabilities,
                 Env.InitTimeoutSec);
             _calculatorSession.Manage().Timeouts().ImplicitWait = Env.ImplicitTimeoutSec;
+            _calculatorSession.LaunchApp();
```

Comment:
shoudln't LaunchApp be marked as deprecated

Reply:
I probably missed the memo, can you please refer me to the topic?

Reply:
@mykola-mokhnach  this one? https://github.com/appium/java-client/pull/1571

Reply:
Removed it anyway